docs/Q-and-A.md

```diff
@@ -0,0 +1,324 @@
+- For the metrics detailed in #3-#6, Bitcoin is so far beyond all other options, it isn't even close - Bitcoin is the most secure option by an absurdly large margin.\
+(_@TheHenryTsai_)
+
+## *Q: How does it work? You need bitcoin transactions for DID, right? So how do BTC transaction fees come into play?
+_Each newly created ID will pay a transaction fee?_
+
+With ION can fit 10,000 ID operations in a single bitcoin transaction.\
+(_@csuwildcat_)
+
+ION doesn't bloat the bitcoin blockchain. It gathers multple proofs of identifier events and anchors them with one single small transaction (and dito fee) on the bitcoin blockchain.\
+(_@henkvancann_)
+
+#### *Q: Who pays the fee if it's all decentralised?
+Node operators who anchor operation batches must pay BTC txn fees. Current we cover the cost for folks who want to anchor via our node, but you can just do it on your own node at home if you want. Ops are signed with keys only you hold, so it's not like another node has your keys.\
+(_@csuwildcat_)
+
+## **@: Can I prune the local bitcoin node?
+I'd use prune on only the non-ion txns, because we haven't yet accounted for that. Totally on the roadmap to fully support a pruned node.\
+(_@csuwildcat_)
+
+## ***Q: What's different between ION and NameID (https://nameid.org) or XID (https://github.com/xaya/xid)? They're on different blockchains, but still based on Bitcoin Core.
+Two key differences:
+1. First and foremost: ION _is not_ another blockchain, it's a Layer 2 protocol that directly leverages Bitcoin's security
+2. It's a massively, provably scalable architecture, unlike those alt-chains\
+(_@csuwildcat_)
+
+# Q&A section ION operational
+
+## *Q: Is ION a coin or an ICO?
+**This has absolutely nothing to do with** that abandoned ICO coin! ION has no token or blockchain of its own.
+
+## **Q:  What would be the specs for an ION node, excluding Bitcoin Core ressources consumption?
+Not much, basically just storage growth as the network is utilized. There's hardly any computation on operation ingest, so it's not going to hit a CPU hard.\
+For example: with 40 billion DIDs in the system (basically all of humanity using ION), it would require around 2-3TB of space, but wouldn't noticeably rev compute on an i3 from 2017.\
+(_@csuwildcat_)
+
+## ***Q: What would be the quickest way for me to learn more about 'unpublished' DIDs? 
+_Do you happen to know of a link to any documentation about that?_
+
+This one class invocation will instantly create you an unpublished DID: [here](https://t.co/eBY4wR2kTk?amp=1)\
+(_@csuwildcat_)
+
+## ***Q: Any plans to make this work on top of bitcoin full nodes?; on top of projects like [Umbrel](https://blocktalk.co/2021/03/25/how-to-set-up-full-bitcoin-node-umbrel/)?
+I haven't had time to sort out getting it packaged for this. Would love some help with that from the community, else it will probably take a couple months before we can dig out.\
+(_@csuwildcat_)
```

Comment:
You could mention that @lounes_kmt is working on an Umbrel implementation: https://github.com/getumbrel/umbrel/issues/560

Reply:
done, thx for tip

Reply:
Can you revert that? It shouldn't be announced officially, sorry.

Reply:
yes of course, and no prob yet, it's still only on my repo

Reply:
has gone